templates/ecommerce/prices/price.yml

```diff
@@ -25,7 +25,7 @@
     <offers>
       {% for option in options %}
         <offer id="{{ option.product.id }}" available="{{ option.in_stock|yesno:'true,false' }}">
-          <name>{{ option.mark }} - {{ option.product.name }} {{ option.specification }}</name>
+          <name>{{ option.mark }} - {{ option.product.name }}</name>
```

Comment:
```suggestion
          <name>{{ option.catalog_name }}</name>
```

Reply:
@artemiy312 , `option.catalog` name is `{{ option.product.name }} {{ option.mark }}`.
It's not equivalent pattern, so we can't use it here